### DISCUSSION TITLE
Which Next.js version, Layouts RFC was released?

### DISCUSSION THREAD

**Discussion by FelipeBattistotti** [vercel/next.js#40826] [Help] [answered]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

Next.js version = latest

### What browser are you using? (if relevant)

Chrome 105.0.5195.127

### How are you deploying your application? (if relevant)

Vercel

### Describe the Bug

Which Next.js version, Layouts RFC was released?

### Expected Behavior

Which Next.js version, Layouts RFC was released?

### Link to reproduction

https://github.com/FelipeBattistotti/devinfo-card-frontend-v2.git

### To Reproduce

Which Next.js version, Layouts RFC was released?

**Comment by ijjk** [comment]
Hi, the layouts RFC here https://github.com/vercel/next.js/discussions/37136 has not yet been published in a Next.js release, please stay tuned.

**Reply by FelipeBattistotti** [reply]
Hi @ijjk ,
Is scheduled to be release in which Next.js version?

I read the blog:
[https://nextjs.org/blog/layouts-rfc](https://nextjs.org/blog/layouts-rfc)

It's possible to test Layouts now?

**Comment by leerob** [comment, marked as answer]
**Update:** We’re excited to announce [Next.js 13](https://nextjs.org/13), which includes the new `app/` directory (beta) with support for nested layouts, colocation of data fetching, streaming, Server Components by default, and much more.

Inside the `app/` directory, there is a powerful new way to fetch data with React’s `use()` hook and the extended Web `fetch()` API. This allows you to colocate data fetching directly inside components for the most flexibility, including inside layouts.

You can leave feedback here: https://github.com/vercel/next.js/discussions/41745
